operator/pkg/operator.go

```diff
@@ -150,9 +150,20 @@ func (o *Operator) ProcessNewTaskCreatedLog(newTaskCreatedLog *servicemanager.Co
 	switch provingSystemId {
 	case uint16(common.GnarkPlonkBls12_381):
 		verificationKey := newTaskCreatedLog.Task.VerificationKey
-		VerificationResult := o.VerifyPlonkProof(proof, pubInput, verificationKey)
+		VerificationResult := o.VerifyPlonkProofBLS12_381(proof, pubInput, verificationKey)
 
-		o.Logger.Infof("PLONK proof verification result: %t", VerificationResult)
+		o.Logger.Infof("PLONK BLS12_381 proof verification result: %t", VerificationResult)
+		taskResponse := &servicemanager.AlignedLayerServiceManagerTaskResponse{
+			TaskIndex:      newTaskCreatedLog.TaskIndex,
+			ProofIsCorrect: VerificationResult,
+		}
+		return taskResponse
+
+	case uint16(common.GnarkPlonkBn254):
+		verificationKey := newTaskCreatedLog.Task.VerificationKey
+		VerificationResult := o.VerifyPlonkProofBN254(proof, pubInput, verificationKey)
```

Comment:
We could change it to:
`verificationResult := o.verifyPlonkProof(proofBytes, pubInputBytes, verificationKeyBytes, ecc.BN254)`
See comment below.